Mobile client

yashpar1 opened the pull request:
package.json updated with app info (name, icon)
App.js now has pages for Logging in, Signing Up, and Signing up to Debate

yashpar1 committed:
Sign Up

Added input fields for sign up page
Set app name and icon

yashpar1 committed:
App.js updated with screens

package.json updated with app name and icon
App.js updated with Log In, Sign Up, and Sign Up to Debate pages
App.js still needs a navigator
App.js still needs headers
App.js still needs a home page and room assignment page

yashpar1 committed:
Dropdown menus to app.js

Dropdown menus added to the "Sign Up to Debate" screen in app.js

yashpar1 commented:
the app.js file now has dropdown menus

shannonjanehogan commented on App.js:
```diff
@@ -1,18 +1,150 @@
+//           <Text style={{padding: 10, fontSize: 42}}>
+//           </Text>
+//         </View>
+//         <View style={{padding: 10}}>
```
check eslint- this would be failing your linter check. Object spacing (should have a space before/after the curly braces on the inside. e.g. `{{ padding: 10 }}`.

yashpar1 replied:
Cool, didn't realize spacing wasn't just a stylistic preference in React I'll fix that

shannonjanehogan commented on App.js:
```diff
@@ -1,18 +1,150 @@
+//           </Text>
+//         </View>
+//         <Button
+//           onPress={() => { Alert.alert('Account Created!')}}
```
We should not be using alerts (generally quite disruptive to the user experience. onPress should call a function called "onLoginPress", which can be empty for now. On successful login, we can just redirect to the sign up to debate page

yashpar1 replied:
sounds good, I'll fix that

shannonjanehogan commented on App.js:
```diff
@@ -1,18 +1,173 @@
+        <View style={{padding: 10}}>
+          <TextInput
+            style={{height: 40}}
+            placeholder="Last Name"
```
Match what is used on the web app- "Sign up to debate"

shannonjanehogan commented on App.js:
```diff
@@ -1,18 +1,173 @@
+//   //   title: 'Sign Up',
+//   // };
+//   constructor(props) {
+//     super(props);
```
is the text actually being used for anything? this should have all of the fields that you are updating- e.g. username, password, and typing on each input should have a corresponding function that updates the state. For an example, you can check the functions/state on the corresponding component on the web side

yashpar1 replied:
It currently isn't doing anything, I'm pretty sure it won't be doing anything in the future either so it'll likely be gone soon.

shannonjanehogan reviewed (changes requested):
Could you add screenshots of your pages in progress? 

I'll approve this PR once the components are in different files. 

Also- this is a really good post on how to write good commit messages. Generally should have a lot more commits :) https://chris.beams.io/posts/git-commit/

yashpar1 commented:
![log in](https://user-images.githubusercontent.com/29381117/28848140-ba289f70-76c6-11e7-90bb-e8e2e74031a2.png)
![sign up to debate](https://user-images.githubusercontent.com/29381117/28848142-ba2e18d8-76c6-11e7-9997-82f3cf0a7a62.png)
![sign up](https://user-images.githubusercontent.com/29381117/28848141-ba2af5cc-76c6-11e7-8861-a5091ffdec26.png)

yashpar1 committed:
Components Reorganized

Components removed form App.js and each put into their own file
Fixes to syntax made
Next step: creat a home screen and navigator

yashpar1 committed:
Made Components folder

Put the different components into their own files, put those files in a folder
Made a Navigator in App.js
The navigator is broken

yashpar1 committed:
Navigator Updated

App.js was edited so that DrawerNavigator functions
Still need to add in navigation between different screens, but the app renders now
